src/Components/Web/src/Web/EventHandlers.cs

```diff
@@ -122,6 +122,9 @@ namespace Microsoft.AspNetCore.Components.Web
     [EventHandler("onpointerlockerror", typeof(EventArgs), true, true)]
     [EventHandler("onreadystatechange", typeof(EventArgs), true, true)]
     [EventHandler("onscroll", typeof(EventArgs), true, true)]
+
+    // Specific events
+    [EventHandler("ontoggle", typeof(EventArgs), true, true)]
```

Comment:
You'll likely have to regenerate ref assemblies: https://github.com/dotnet/aspnetcore/blob/master/docs/ReferenceAssemblies.md#when-changing-public-api